rebar.config

```diff
@@ -18,5 +18,6 @@
   {pbkdf2, ".*", {git, "git://github.com/basho/erlang-pbkdf2.git", {tag, "2.0.0"}}},
   {eleveldb, ".*", {git, "git://github.com/basho/eleveldb.git", {tag, "ts-1.0.4"}}},
   {exometer_core, ".*", {git, "git://github.com/basho/exometer_core.git", {tag, "1.0.0-basho6"}}},
+  {jam, ".*", {git, "git://github.com/macintux/jam.git", {branch, "master"}}},
```

Comment:
@macintux mentioned this in HipChat.  We'll try to get the repo copied over to Basho, but not in this PR.